src/otx/recipe/detection/yolox_tiny.yaml

```diff
@@ -38,23 +38,23 @@ overrides:
   gradient_clip_val: 35.0
   data:
     input_size:
-      - 640
-      - 640
+      - 416
+      - 416
     train_subset:
       batch_size: 8
       transforms:
         - class_path: otx.core.data.transform_libs.torchvision.CachedMosaic
           init_args:
             random_pop: false
             max_cached_images: 20
-            img_scale: $(input_size) # (H, W)
+            img_scale: $(input_size) * 1.538 # 640x640
```

Comment:
Why do we need this trick? Can we just declare 640 as input?

Reply:
input_size is used when exporting a model and yolox_tiny uses 416 for validation and test subset. So, input_size should be set based on val or test input size.